src/styles/main.css

```diff
@@ -0,0 +1,110 @@
+.page {
```

Comment:
[Checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) the 11th point